homework-submission/React-Week3/src/todoapp/TodoItem.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import '../App.css';
+
+class TodoItem extends Component {
+    render() {
+        const { id, description, deadline, done } = this.props.todo;
+        const { clickHandler } = this.props;
+        const { removeHandler } = this.props;
+        return (
+            < li className={done ? "done" : "un-done"} >
+                <input type="checkbox" id={id} name="Todo" onChange={clickHandler} checked={done} />
```

Comment:
You have to chose a better name rather than `clickHandler`, like `checkDoneToggle` may be !! and it make sense because it's a toggle and for the done and checkbox, sure you have a better one 😃.